analysis/_site.yml

```diff
@@ -1,25 +1,28 @@
 name: fucci-seq
-output_dir: "../docs"
-include: [".nojekyll"]
-exclude: ["include"]
+output_dir: ../docs
+include: .nojekyll
+exclude: include
```

Comment:
At this point I'm just going to leave these changes that are getting automatically made to `_site.yml`. I'm not sure how it is happening, but I don't think it should cause a problem.